app/extended_modules/keythereum/exports.js

```diff
@@ -1,2 +0,0 @@
-var keythereum = global.keythereum || require('./');
```

Comment:
we are bringing back this custom module since it breaks keystore unlock without it - added in other branch